### PR TITLE
Add a list of mojaloop repos in categories

### PR DESCRIPTION
**Pull request by lewisdaly**
- part of: https://github.com/mojaloop/project/issues/948

### CODE REVIEW

File: repositories/README.md [hunk 1]
```diff
@@ -2,14 +2,59 @@
...
+- [central-services-stream](https://github.com/mojaloop/central-services-stream)
+- [central-services-metrics](https://github.com/mojaloop/central-services-metrics)
+- [central-object-store](https://github.com/mojaloop/central-object-store)
+- [event-sdk](https://github.com/mojaloop/event-sdk)
+- [pathfinder-query-client](https://github.com/mojaloop/pathfinder-query-client)
+- [pathfinder-provisioning-client](https://github.com/mojaloop/pathfinder-provisioning-client)
+- [sdk-scheme-adapter](https://github.com/mojaloop/sdk-scheme-adapter)
+- [sdk-standard-components](https://github.com/mojaloop/sdk-standard-components)
+
+### Experimental
```
Comment (elnyry) [inline repositories/README.md]: I think the first 5 here are now past the experimental phase and into snapshot phase :-)

File: repositories/README.md [hunk 2]
```diff
@@ -2,14 +2,59 @@
...
+- [central-services-shared](https://github.com/mojaloop/central-services-shared)
+- [central-services-database](https://github.com/mojaloop/central-services-database)
+- [central-services-health](https://github.com/mojaloop/central-services-health)
+- [central-services-stream](https://github.com/mojaloop/central-services-stream)
+- [central-services-metrics](https://github.com/mojaloop/central-services-metrics)
+- [central-object-store](https://github.com/mojaloop/central-object-store)
+- [event-sdk](https://github.com/mojaloop/event-sdk)
+- [pathfinder-query-client](https://github.com/mojaloop/pathfinder-query-client)
+- [pathfinder-provisioning-client](https://github.com/mojaloop/pathfinder-provisioning-client)
+- [sdk-scheme-adapter](https://github.com/mojaloop/sdk-scheme-adapter)
```
Comment (elnyry) [inline repositories/README.md]: I think sdk-scheme-adapter is a service in itself, may be can be moved to QA / Tools along with mojaloop-simulator

File: repositories/README.md [hunk 3]
```diff
@@ -2,14 +2,59 @@
...
+- [account-lookup-service](https://github.com/mojaloop/account-lookup-service)
+- [bulk-api-adapter](https://github.com/mojaloop/bulk-api-adapter)
+- [central-event-processor](https://github.com/mojaloop/central-event-processor)
+- [central-ledger](https://github.com/mojaloop/central-ledger)
+- [central-settlement](https://github.com/mojaloop/central-settlement)
+- [email-notifier](https://github.com/mojaloop/email-notifier)
+- [forensic-logging-client](https://github.com/mojaloop/forensic-logging-client)
+- [forensic-logging-sidecar](https://github.com/mojaloop/forensic-logging-sidecar)
+- [ml-api-adapter](https://github.com/mojaloop/ml-api-adapter)
+- [helm](https://github.com/mojaloop/helm)
```
Comment (elnyry) [inline repositories/README.md]: Helm --> Tools?

File: repositories/README.md (hunk 1 above)
Comment (lewisdaly) [reply repositories/README.md]: Ah cool. Do they belong in a separate snapshot section? Or somewhere else?

Comment (elnyry) [reply repositories/README.md]: @lewisdaly I think we do need a Snapshot section and yes Kim, the repos should have appropriate descriptions.. For example apart from normal ones, we need to use DEPRECATED, SNAPSHOT, EXPERIMENTAL as a prefix to descriptions as relevant, I think

Comment (lewisdaly) [reply repositories/README.md]: Cool. I'll add `Update the repo descriptions` to the original ticket, and include a starting "tag" as above in those descriptions.

**Comment by lewisdaly** [comment]
@elnyry @kjw000 I had another shot at recategorizing them, let me know what you think.

File: repositories/README.md (hunk 3 above)
Comment (elnyry) [reply repositories/README.md]: @lewisdaly can you please revisit this?

File: repositories/README.md [hunk 4]
```diff
@@ -2,14 +2,65 @@
...
 * The [project repository](https://github.com/mojaloop/project) is the central repository to track product development issues for the Mojaloop project.
 * The [mojaloop-specificiation](https://github.com/mojaloop/mojaloop-specification) contains the specification document set of the Open API for Financial Service Provider Interoperability.
 
-Individual repositories in the [Mojaloop GitHub organization](https://github.com/mojaloop/) each describe component-specific details including source and APIs.
+## Repository List
+
+Individual repositories in the [Mojaloop GitHub organization](https://github.com/mojaloop/) each describe component-specific details including source and APIs. Below is a non-exhaustive list of the commonly used Mojaloop repositories, grouped into relevant categories.
+
+### Core Related Services
+- [account-lookup-service](https://github.com/mojaloop/account-lookup-service)
```
Comment (elnyry) [inline repositories/README.md]: Can we call them "Core services" and then just have ALS, bulk-api-adapter, central-ledger, central-settlement, ml-api-adapter? Helm can be under tools.. CEP, email-notifier, event-sidecar, FLS, etc can be core-related services..

Please let me know what you think.

Comment (lewisdaly) [reply repositories/README.md]: Sure that sounds good

**Pull request merged**